client/src/styles/colors.module.scss

```diff
@@ -0,0 +1,6 @@
+$c-primary: #1d3557;
+$c-secondary: #457b9d;
+$c-terciary: #a8dadc;
+$c-background: #f1faee;
+$c-contrast: #EA483D;
+$c-text: #191919;
```

Comment:
Nice that you added colors as variables! Just fyi, in bigger projects it is mostly common to have variables for different shades of a color palette, which are called e.g. `grey.10` etc. see [tailwind colors](https://tailwindcss.com/docs/customizing-colors).

But this will totally work out for this project and having abstract variable names instead of fixed colors works as well since you might want to change the colors later on.  There are possibilities to combine the best of both but that would be quite some overhead, so just fyi